timelapse_microscopy/timelapse_microscopy.ipynb

```diff
@@ -0,0 +1,1187 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7d9a24ab-8ff1-4595-9246-54bc0afb03bf",
+   "metadata": {},
+   "source": [
+    "## App V2: Intermediate Viewer with Side Views"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "62e45d5d-af38-4538-be52-2304ed119012",
+   "metadata": {},
+   "source": [
+    "As our data array is a three-dimensional volume, let's create a more advanced application using `HoloViews` in place of `hvPlot` to handle the added complexity, and `Panel` for more control over the layout and interactive links.\n",
+    "\n",
+    "This more advanced app builds on the previous one with added functionality, such as.\n",
+    "\n",
+    "1. **Side Views**: Aggregated side views for display over 'deep' time dimension.\n",
+    "2. **Synchronized Frame Indicators**: Frame markers on the Side Views synchronized with the playback and x,y range of the main image stack view.\n",
+    "4. **Slider Overlay Opacity**: Slider widget to adjust opacity of max-over-time overlay for direct comparison and a tighter layout.\n",
+    "5. **Scale Bar**: A dynamic and customizable visual reference for spatial scale.\n",
+    "3. **Continuous Playback**: Player widget for continuous playback, along with controls for step-by-step examination of the image stack."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a54065a9-9389-4c21-a020-743bdb3f2f0f",
+   "metadata": {},
+   "source": [
+    "### Main Image and Playback Controls"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a263ad4b-6eee-4399-9eb6-278d7cc12205",
+   "metadata": {},
+   "source": [
+    "\n",
+    "<img \n",
+    "    src=\"./assets/int_app_main_w_player.png\"\n",
+    "    alt=\"Static Preview Intermediate app Main w Player\" \n",
+    "    align=\"right\"\n",
+    "    width=70%\n",
+    "    >\n",
+    "\n",
+    "**Here's a static snapshot of the main view and frame player that we'll make now. 👉**\n",
+    "\n",
+    "\n",
+    "<div style=\"clear: both;\"></div>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fbd68bc1-d90c-4c34-a5dc-c6cf5781b5ab",
+   "metadata": {},
+   "source": [
+    "First, we'll define a function to create the main image view for a given frame."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e25fbda1-bf0a-47cd-8f21-8cbc9a095e34",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def plot_image(frame):\n",
+    "    return hv.Image(da.sel(frame=frame), kdims=[\"width\", \"height\"]).opts(\n",
+    "        title=f'Frame = {frame}',\n",
+    "        frame_height=da.sizes['height'],\n",
+    "        frame_width=da.sizes['width'],\n",
+    "        cmap='Viridis',\n",
+    "        clim=(0, 20),\n",
+    "        colorbar=True,\n",
+    "        tools=['hover', 'crosshair'],\n",
+    "        toolbar='right',\n",
+    "        apply_hard_bounds=True,\n",
+    "        scalebar=True,\n",
```

Comment:
There's no scalebar displayed at all in fact? Bug maybe?

Reply:
waiting for [Bokeh 3.6.2](https://github.com/bokeh/bokeh/issues/14153)

Reply:
reminder to self to relock when bokeh pin updated

Reply:
added as todo in descr to unblock rebase on main